<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>petprofile</class>
 <widget class="QWidget" name="petprofile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="petName">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>36</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pet Profile</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="pet_image">
     <property name="minimumSize">
      <size>
       <width>175</width>
       <height>175</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1600</width>
       <height>1600</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Picture Goes Here</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PB_switchImg">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>161111</width>
       <height>17</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #f0f0f0;
}</string>
     </property>
     <property name="text">
      <string>Switch Image</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="petInfo">
        <property name="font">
         <font>
          <family>Manjari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Pet Name, Age, Weight</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="petSpecies">
        <property name="font">
         <font>
          <family>Manjari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Species</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="petBreed">
        <property name="font">
         <font>
          <family>Manjari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Breed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="petGender">
        <property name="font">
         <font>
          <family>Manjari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Gender</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="petOther">
        <property name="font">
         <font>
          <family>Manjari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Rescued or Diseased?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="prevPref">
        <property name="font">
         <font>
          <family>Manjari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Previous Preference</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="prefStatus">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>--Select your new preference--</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dislike</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Neutral</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Like</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="adoptionStatus">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Adoption Status: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="request">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 25px;
    background-color: #ffb6c1;
}</string>
       </property>
       <property name="text">
        <string>Request
 Adoption</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="update">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 25px;
    background-color: #00C0ff;
}</string>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lastViewDT">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>15</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Last Viewed: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
